tests/e2e/controlplane/control_plane_suite_test.go

```diff
@@ -38,6 +38,7 @@ var (
 	istioCniName          = env.Get("ISTIOCNI_NAME", "default")
 	image                 = env.Get("IMAGE", "quay.io/maistra-dev/sail-operator:latest")
 	skipDeploy            = env.GetBool("SKIP_DEPLOY", false)
+	expectedRegistry      = env.Get("EXPECTED_REGISTRY", "^docker\\.io|^gcr\\.io")
```

Comment:
maybe add this new variable to tests/e2e/README?

Reply:
done